authorize_test.go

```diff
@@ -213,11 +217,11 @@ func TestAutographerAddAuthorizationsFails(t *testing.T) {
 			name: "authorization without a signer ID fails",
 			auths: []authorization{
 				{
-					ID:      "alice",
+					ID:      "bernie",
```

Comment:
Changed ID so we could use the same `autograper` for each test case rather than recreating it to prevent collisions.